src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -328,6 +363,7 @@ export default () => {
   const { refetch, loading, error, data } = useQuery(workspaces)
   const [create, redirectId] = useCreate()
   const [_delete] = useDelete()
+  const [_clone] = useClone()
```

Comment:
:pencil2: Any specific reason to add the `_` to the variable name? We had to do that with delete because it's a reserved keyword

Reply:
no reason, will update